config/config_params.go

```diff
@@ -135,6 +135,11 @@ type Config struct {
 	TyphaReadTimeout    time.Duration `config:"seconds;30;local"`
 	TyphaWriteTimeout   time.Duration `config:"seconds;10;local"`
 
+	// Configure where Felix gets its routing information.
+	// - workloadIPs: use workload endpoints to construct routes.
+	// - calicoIPAM: use IPAM data to contruct routes.
+	RouteSource string `config:"oneof(workloadIPs,calicoIPAM);calicoIPAM"`
```

Comment:
Maybe we should align new params with libcalico-go naming?  I.e `WorkloadIPs` (c.f. `Allow|Deny` in policies)?